minikube-nfs.sh

```diff
@@ -309,9 +309,6 @@ sudo umount -i /Users'
 sudo mkdir -p \""$shared_folder"\""
   done
 
-  bootlocalsh="${bootlocalsh}
-sudo /usr/local/etc/init.d/nfs-client start"
```

Comment:
This is required for the new nfs settings to take effect in case the VM is rebooted

Reply:
Never mind, this file does not exist in a minikube VM
```bash
$ ls /usr/local/etc/init.d/nfs-client
ls: cannot access '/usr/local/etc/init.d/nfs-client': No such file or directory
```